python/lsst/ts/observatory/control/base_calsys.py

```diff
@@ -89,7 +89,9 @@ async def setup_calsys(self, calib_type: str) -> None:
         The dome and telescope can be moving into place concurrently.
 
         """
-        raise NotImplementedError()
+        await self.warmup_lightsource(calib_type)
+        # Move dome into place?
```

Comment:
No, this is not the job of the CalSys

Reply:
Ok. That's why I made a new method of warmup_lightsource in case we wanted to add other things.